pages/docs/reference/js-project-setup.md

```diff
@@ -365,7 +365,21 @@ The Kotlin/JS Gradle plugin automatically generates a standard webpack configura
 at build time which you can find the at `build/js/packages/projectName/webpack.config.js`.
 
 The most common webpack adjustments can be made directly via the
-`kotlin.js.browser.webpackTask` configuration block in the Gradle build file.
+`kotlin.js.browser.webpackTask` configuration block in the Gradle build file. These are:
```

Comment:
When I first read the sentence, I didn't get what `These are` relate to. After reading twice, I understood. But I am not sure you need this phrase and it adds value. You can just put the semicolon after the first sentence:
> The most common webpack adjustments can be made directly via the
`kotlin.js.browser.webpackTask` configuration block in the Gradle build file: